R/models.R

```diff
@@ -47,7 +48,7 @@ cloudml_deploy <- function(
                   ("models")
                   ("create")
                   (name)
-                  ("--regions=%s", gcloud$region))
+                  ("--regions=%s", region))
```

Comment:
How does `region` acquire a value if the user passes `NULL` (it looks like we only set `gcloud$region` above)

Reply:
When `NULL`, we avoid passing a parameter, this is the behavior we want, right?

This works since `MLArgumentsBuilder` written by Kevin skips parameters that are `NULL`, for instance:

```
> cloudml:::MLArgumentsBuilder(list())("--regions=%s", "foo")()
[1] "--account"          "javier@rstudio.com" "--project"          "rstudio-cloudml"   
[5] "ml-engine"          "--regions=foo"     
> cloudml:::MLArgumentsBuilder(list())("--regions=%s", NULL)()
[1] "--account"          "javier@rstudio.com" "--project"          "rstudio-cloudml"   
[5] "ml-engine"  
```

Reply:
Oh, this is for `cloudml_deploy()`, good catch, we do need a similar change to `cloudml_train()`, fixed.